### DISCUSSION TITLE
how to access a serivce in namespace with istio enabled from a name sapce with istio disabled?

### DISCUSSION THREAD

**Discussion by ogolan-ig** [istio/istio#52211] [Q&A]
hi, 

i a service in namesapce stage that has istio injection enabled. i can do curl scraper-base-service.stage.svc.cluster.local from a pod inside stage namesapce sucessfully. but trying to do the same from a pod in another namespace with istio disabled results in curl: (56) Recv failure: Connection reset by peer.

im guessing istio is blocking me. can i workaround this?

**Comment by zirain** [comment]
did you enable strict mTLS?

**Comment by ogolan-ig** [comment]
Yes

בתאריך יום ב׳, 22 ביולי 2024, 3:23, מאת zirain ***@***.***>:

**Reply by zirain** [reply]
I assume it worked as expected.

**Comment by ogolan-ig** [comment]
no i meant i enabled  strict mTLS but i want this specific http call not to
use certificate, is this possible?

**Reply by sridhargaddam** [reply]
I think you have two options:

1. You can configure PeerAuthentication spec.mtls.mode as `PERMISSIVE` (if its acceptable for you)
2. Explore using PeerAuthentication spec.portLevelMtls and disable mTLS for specific ports as documented here - https://istio.io/latest/docs/reference/config/security/peer_authentication/
